	Glibc-2.16 sys/wait.h no longer includes sys/resource.h, and that
	causes a problem with some busybox programs not having rlim_t (or
	some other resource definitions) defined; so add an include of
	sys/resource.h, which probably should have already been there.
	2012-12-15 <douglas@ttylinux.net>

diff -Naur busybox-1.20.2-OLD/loginutils/passwd.c busybox-1.20.2-NEW/loginutils/passwd.c
--- busybox-1.20.2-OLD/loginutils/passwd.c	2012-06-26 06:35:45.000000000 -0700
+++ busybox-1.20.2-NEW/loginutils/passwd.c	2012-12-22 09:57:29.901004274 -0700
@@ -15,6 +15,7 @@
 
 #include "libbb.h"
 #include <syslog.h>
+#include <sys/resource.h>
 
 static void nuke_str(char *str)
 {
diff -Naur busybox-1.20.2-OLD/networking/inetd.c busybox-1.20.2-NEW/networking/inetd.c
--- busybox-1.20.2-OLD/networking/inetd.c	2012-07-02 07:08:25.000000000 -0700
+++ busybox-1.20.2-NEW/networking/inetd.c	2012-12-22 09:57:06.651282966 -0700
@@ -165,6 +165,7 @@
 //usage:     "\n		(default: 0 - disabled)"
 
 #include <syslog.h>
+#include <sys/resource.h>
 #include <sys/un.h>
 
 #include "libbb.h"
